README.md

```diff
@@ -2,13 +2,34 @@
 Guidelines on how we write and structure iOS projects at Wolfpack-Digital
 
 
-1. Structure of the project folders
-    1. AppDelegate - for AppDelegate.swift and extensions of AppDelegate
-    2. ViewControllers - for view controllers and extensions of view controllers (tableview cells can go in here as well)
-    3. Models - for models
-    4. Lib - for backend logic
-    5. Extensions - for extensions of external classes - if you for example extend a view controller of your own, put it in the ViewControllers
-    6. SupportingFiles - for Info.plist and other files
+1. Structure of the project folders. The structure follows some rules: high level code top -> low level code bottom. If the layer is the same, use user flow order, if user flow doesn't matter, use alpanumeric ordering. Use the example bellow as a starting point.
+    e.g.:
+    - `App` - for AppDelegate, extensions of AppDelegate and global clases that are supposed to be used at entry level of the app
+    - `Navigation` - for coordinators and other logic related to navigation (e.g. deeplinks)
+    - `Presentation` - for view controllers and extensions of view controllers
+        - `Launch` - for launch and app start loading UI
+        - `Authentification` - Login, Signup related UI
+            - `Welcome`
+                - `...`
+            - `Login`
+                - `LoginViewController`
+                    - `LoginViewController.xib` - `.xib` file above its counterpart `.swift` file
+                    - `LoginViewController.swift`
+                    - `LoginViewModel.swift` - viewModel bellow the view controller
+                - `Cells` - cells used only for the login screen
+            - `Signup`
+                - `...`
+        - `Main` - main screens of the app. Add more levels of subgroupping if needed.
+        - `Cells` - cells supposed to be used throughout the app
```

Comment:
I think we also need here a folder for Storyboards, for the projects that use storyboards.

Reply:
Good point, I will add it. I believe we should keep storyboards to a limited use, as they are not very reusable especially when using segues. If they are used only for grouping the screens and implement the navigation from code, I think would be ok.

Reply:
Yes, we can see how much we need to reuse each view in each project. Most of them have dedicated screens for specific functionalities, so we don't need to actually reuse grouped UI components. When we do need to reuse them like that, I agree that xibs are a better option, but this might not happen that often.